WpfApp4/WpfApp4/Model/Countries.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WpfApp4.Model
+{
+    public static class Countries
```

Comment:
It seems that this class is helper for countries... Instead of Enum...
Should be placed in appropriate folder.

Reply:
Also name should be changed to describe what it provides.